packages/puppeteer-core/src/common/JSHandle.ts

```diff
@@ -72,7 +46,7 @@ export interface BoundingBox extends Point {
  *
  * @public
```

Comment:
I think we can mark implementation class and methods as internal. We can probably remove duplicated documentation in the implementation class too (I think docs diff is failing because of that)